compiler/test/suites/boxes.re

```diff
@@ -0,0 +1,64 @@
+open TestFramework;
+open Runner;
+
+describe("boxes", ({test}) => {
+  let assertSnapshot = makeSnapshotRunner(test);
+  let assertCompileError = makeCompileErrorRunner(test);
+  let assertRun = makeRunner(test);
+  let assertFileRun = makeFileRunner(test);
+
+  assertRun(
+    "box1",
+    "let b = box(4);\n            {\n              print(unbox(b))\n            }",
```

Comment:
Can we fix the weird spacing on these?

Reply:
Nah, I wanted to change the existing tests as little as possible in this PR. This was already planned to be fixed in #479.